src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractDescriptionBuilder.cs

```diff
@@ -388,6 +388,21 @@ private void BuildXmlDocumentation(ISymbol symbol, Compilation compilation, _VSO
                 emittedDocs = true;
             }
 
+            if (ShowValueDocumentation(symbol) && documentationComment.ValueText != null)
```

Comment:
Is it possible to include both a `returns` and `value` in the doc comments?

Reply:
Yes, it's possible.